contracts/adapters/FlexibleLeverageStrategyAdapter.sol

```diff
@@ -469,6 +612,39 @@ contract FlexibleLeverageStrategyAdapter is BaseAdapter {
         return _calculateCurrentLeverageRatio(currentLeverageInfo.collateralValue, currentLeverageInfo.borrowValue);
     }
 
+    /**
+     * Calculates the total notional rebalance size. Note: this function does not take into account timestamps, so it may return a nonzero value
```

Comment:
Add a bit more context. `Calculates the total notional rebalance size that can be used by external contracts or keeper bots to determine optimal exchange to rebalance on`